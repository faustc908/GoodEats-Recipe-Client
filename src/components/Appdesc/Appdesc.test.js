import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import AppDesc from './AppDesc'
import ReactDOM from 'react-dom';

describe(`AppDesc component`, () => {

    it('renders a AppDesc by default', () => {
        const wrapper = shallow(<AppDesc />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<AppDesc />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
})