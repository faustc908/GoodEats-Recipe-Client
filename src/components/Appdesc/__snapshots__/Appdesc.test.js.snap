// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppDesc component renders a AppDesc by default 1`] = `
<div>
  <img
    alt=""
    className="banner-image"
    src="https://images.squarespace-cdn.com/content/v1/5b4e0e6a5b409b5848de216a/1536382993388-YJ6HC0RY1XHMX1EH88FD/ke17ZwdGBToddI8pDm48kAiQauKMq5t3zNDTFOhTnlJ7gQa3H78H3Y0txjaiv_0fDoOvxcdMmMKkDsyUqMSsMWxHk725yiiHCCLfrh8O1z4YTzHvnKhyp6Da-NYroOW3ZGjoBKy3azqku80C789l0tb-hnCqoepq4X8c1traqO-_NrqY7vRbriOxzaFgT6Y6dynDdRL5TMOOSto-dGo7jQ/smart-food-banner.jpg?format=2500w"
  />
  <p
    className="desc"
  >
    Are you tired of the same boring meals day after day? GoodEats helps you find new recipes to try out. Simply enter in an ingredient or dish and find dozens of results! Impress your friends and family with fresh new ideas! If you have a great recipe that you would like to share please add it to the form below!
  </p>
</div>
`;
